process/cg-transition.html

```diff
@@ -106,50 +106,39 @@ <h3>
+      <li>Have those contributors (or the organizations they represent) joined the Working Group or do they plan to do so?</li>
     </ul>
-    <p>
-      By comparison, under the W3C Patent Policy for Working Groups,
-      participants make full specification commitments upon joining the group.
-    </p>
-    <p id="ipr-commitments">
-      There are thus three ways to procure commitments by Community Group
-      contributors to the full text of a specification transferred to a Working
-      Group:
+
+    <p id="ipr-commitments">If significant contributors have not
+      joined the Working Group or don't plan to, then steps should be
+      taken to secure IPR commitments from them. There are three ways for CG
+      contributors to make commitments for the specification managed
+      by the Working Group:
     </p>
     <ol>
-      <li>They join (or already participate in) the Working Group, or
-      </li>
-      <li>They make a commitment under the Final Specification Agreement.
-      <strong>Note:</strong> FSA commitments may still have some value for
-      situations where a Working Group ultimately does not advance the
-      specification on the Recommendation Track.
+      <li>They can join (or already participate in) the Working Group.</li>
+      <li>They can sign the non-participant licensing agreement for the Working Group.</li>
+      <li>They can make a commitment (via <a href=
+      "https://www.w3.org/community/about/faq/#fsa-mechanics">tools</a>) to the Community Group specification under the <a href=
+      "https://www.w3.org/community/about/agreements/final/">W3C Community
+      Final Specification Agreement (FSA)</a>. Note that the main difference between the FSA and the <a href=
+      "https://www.w3.org/community/about/process/cla/">W3C Community
+      Contributor Licensing Agreement (CLA)</a> is that patent licensing commitments under the CLA are for one's own contributions, while commitments under the FSA are for the full text of the specification.
       </li>
-      <li>They sign the non-participant licensing agreement for the Working Group</li>
     </ol>
-    <p>
+
+      <p><strong>Note:</strong> FSA commitments may still have some
+      value for situations where a Working Group ultimately does not
+      advance the specification on the Recommendation Track.  
       Each Community Group should make a determination whether to launch the
       process of gathering Final Specification commitments, based on
-      (anticipated or actual) Contributor participation in the Working Group. The Community Group Web
-      site includes <a href=
-      "https://www.w3.org/community/about/faq/#fsa-mechanics">tools</a> to
-      request and collect "Final Spec" commitments.
-    </p>
+      (anticipated or actual) Contributor participation in the Working Group. 
+      </p>
```

Comment:
I think that paragraph could be removed altogether; people interested in the value of FSA outside of transition to a WG aren't likely to be reading that page